app/analysis/edit/route.js

```diff
@@ -1,3 +1,9 @@
 import Route from "@ember/routing/route";
 
-export default class AnalysisEditRoute extends Route {}
+export default class AnalysisEditRoute extends Route {
+  resetController(controller, isExiting, transition) {
+    if (isExiting && transition.targetName !== "error") {
+      controller.send("resetController");
```

Comment:
Isnt it possible to call `controller.resetQueryParams()`

Reply:
:bulb: good one!

Reply:
`done`